ocdskingfisherprocess/web/app.py

```diff
@@ -51,11 +51,39 @@ def api_v1():
     return "OCDS Kingfisher APIs V1"
 
 
+@app.route("/api/v1/submit/end_collection_store/", methods=['POST'])
+def api_v1_submit_end_collection_store():
+    api_key = request.headers.get('Authorization', '')[len('ApiKey '):]
+    if not api_key or api_key not in config.web_api_keys:
+        return "ACCESS DENIED", 401
```

Comment:
Would really like the ApiKey check to be in one place and not repeated in every API fuction. Partly to save repetition but mainly to make sure all the logic is the exactly the same especially as its security related.

Reply:
Will update now - that's a quick one